lib/cfnvpn/templates/lambdas/auto_route_populator/quotas.py

```diff
@@ -0,0 +1,37 @@
+import boto3
+
+ROUTE_TABLE_QUOTA_CODE = 'L-401D78F7'
+AUTH_RULE_TABLE_QUOTA_CODE = 'L-9A1BC94B'
```

Comment:
I am assuming these are non-unique values that can be used across accounts?

Reply:
these are AWS quota codes for their specific quota. These values should never change, could of used the API to look them up but it was about 3 APIs deep to get the value. Didn't want to waste valuable lambda execution time to get a static value